charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -43,4 +43,8 @@ spec:
         runAsGroup: 2000
         runAsNonRoot: true
         runAsUser: 2000
+      {{- if .Values.tolerations }}
```

Comment:
is there a specific use-case for toleration for short living jobs?

Reply:
Without tolerations, even if is a short lived job, it will not deploy.

This is code is an extract taken from the manifest file generated around a 8 months ago that I had to modify to make it run in our cluster

```
---
# Source: nri-bundle/charts/nri-metadata-injection/templates/job.yaml

apiVersion: batch/v1
kind: Job
metadata:
  name: nri-bundle-nri-metadata-injection-job
  namespace: newrelic
  labels:
    app.kubernetes.io/name: nri-metadata-injection
    helm.sh/chart: nri-metadata-injection-1.1.2
    app.kubernetes.io/instance: nri-bundle
    app.kubernetes.io/version: "1.3.0"
spec:
  template:
    metadata:
      labels:
        app.kubernetes.io/name: nri-metadata-injection
        helm.sh/chart: nri-metadata-injection-1.1.2
        app.kubernetes.io/instance: nri-bundle
        app.kubernetes.io/version: "1.3.0"
    spec:
      serviceAccountName: nri-bundle-nri-metadata-injection
      containers:
        - name: nri-metadata-injection-job
          # This is a minimal kubectl image based on Alpine Linux that sings certificates using the k8s extension api server
          image: "newrelic/k8s-webhook-cert-manager:1.3.0"
          imagePullPolicy: "IfNotPresent"
          command: ["./generate_certificate.sh"]
          args:
            - "--service"
            - nri-bundle-nri-metadata-injection
            - "--webhook"
            - nri-bundle-nri-metadata-injection
            - "--secret"
            - nri-bundle-nri-metadata-injection
            - "--namespace"
            - newrelic
       # Added the tolerations to make this work
      tolerations:
        - effect: NoSchedule
          operator: Exists
        - effect: NoExecute
          operator: Exists
      restartPolicy: Never

  backoffLimit: 1
```

Reply:
Yes, we have all our of nodes tainted, in our infra can't deploy unless the deployment, job, etc has a taint